org.gweatherrouting.gweatherrouting.yml

```diff
@@ -0,0 +1,162 @@
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+command: gweatherrouting
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --socket=fallback-x11
+  - --socket=wayland
+
+
+modules:
+
+  - name: proj
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+      - -DBUILD_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://github.com/OSGeo/PROJ/releases/download/9.7.1/proj-9.7.1.tar.gz
+        sha256: 6c097dc803c561929cdfcc46e4bf9945ea977611fb31493ad14e88edaeae260f
+
+  - name: geos
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTING=OFF
+    sources:
+      - type: archive
+        url: http://download.osgeo.org/geos/geos-3.13.1.tar.bz2
+        sha256: df2c50503295f325e7c8d7b783aca8ba4773919cde984193850cf9e361dfd28c
+        x-checker-data:
+          type: anitya
+          project-id: 13493
+          url-template: http://download.osgeo.org/geos/geos-$version.tar.bz2
+          stable-only: true
+
+  - name: json-c
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTING=OFF
+      - -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+    sources:
+      - type: git
+        url: https://github.com/json-c/json-c
+        tag: json-c-0.18-20240915
+        commit: 41a55cfcedb54d9c1874f2f0eb07b504091d7e37
+        x-checker-data:
+          type: git
+          tag-pattern: '^json-c-([\d.]+)-[\d]+$'          
+
+  - name: libgeotiff
+    buildsystem: autotools
+    sources:
+    - type: archive
+      url: https://github.com/OSGeo/libgeotiff/releases/download/1.7.4/libgeotiff-1.7.4.tar.gz
+      sha256: c598d04fdf2ba25c4352844dafa81dde3f7fd968daa7ad131228cd91e9d3dc47
+      
+  - name: qhull
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+      - -DBUILD_STATIC_LIBS=OFF
+      - -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+    sources:
+      - type: archive
+        url: https://github.com/qhull/qhull/archive/v8.0.2/qhull-8.0.2.tar.gz
+        sha256: 8774e9a12c70b0180b95d6b0b563c5aa4bea8d5960c15e18ae3b6d2521d64f8b
+
+  - name: lerc
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/Esri/lerc/archive/refs/tags/v4.0.0.tar.gz
+        sha256: 91431c2b16d0e3de6cbaea188603359f87caed08259a645fd5a3805784ee30a0
+        
+  - name: muparser
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/beltoforion/muparser/archive/refs/tags/v2.3.5.tar.gz
+        sha256: 20b43cc68c655665db83711906f01b20c51909368973116dfc8d7b3c4ddb5dd4
+
+  - name: swig
```

Comment:
if this is a build only dependency then it should have a local `"*"` cleanup rule.